util-core/src/test/scala/com/twitter/util/TimeTest.scala

```diff
@@ -307,19 +302,36 @@ trait TimeLikeSpec[T <: TimeLike[T]] extends WordSpec with GeneratorDrivenProper
     }
 
     "deal with undefineds" in {
-      assert(Bottom.floor(1.second) === Bottom)
       assert(Undefined.floor(0.seconds) === Undefined)
       assert(Undefined.floor(Duration.Top) === Undefined)
       assert(Undefined.floor(Duration.Bottom) === Undefined)
       assert(Undefined.floor(Duration.Undefined) === Undefined)
     }
 
-    "floor itself" in {
+    "round to itself" in {
       for (s <- Seq(Long.MinValue, -1, 1, Long.MaxValue); t = fromNanoseconds(s))
         assert(t.floor(Duration.fromNanoseconds(t.inNanoseconds)) === t)
     }
   }
 
+  "floor" should {
+    "round down" in {
```

Comment:
feel free to ignore:
I think this is a tiny bit more readable if you use 60.seconds helpers rather than fromSeconds, but it's not a big deal (and fromSeconds will actually make it compile faster, so there's definitely an argument the other way too).